vault/vault-operator-values.yaml

```diff
@@ -1,15 +1,16 @@
 defaultVaultConnection:
-  # toggles the deployment of the VaultAuthMethod CR
   enabled: true
-
-  # Address of the Vault Server
-  # Example: http://vault.default.svc.cluster.local:8200
   address: "http://vault.vault.svc.cluster.local:8200"
   skipTLSVerify: false
-  spec:
-  template:
-    spec:
-      containers:
-      - name: manager
-        args:
-        - "--client-cache-persistence-model=direct-encrypted"
\ No newline at end of file
+controller:
+  manager:
+    clientCache:
+      persistenceModel: direct-encrypted
+      storageEncryption:
+        enabled: true
+        mount: kubernetes
+        keyName: vso-client-cache
+        transitMount: demo-transit
+        kubernetes:
+          role: auth-role-operator
+          serviceAccount: default
```

Comment:
The best security practice is to create a unique serviceAccount for this setup. Using the `default` serviceAccount can be considered insecure. 

I think we can document the use of VSO's own serviceAccount rather than default: 
E.g: (will need to adapt for the VSO namespace)
```shell
kubectl get serviceaccounts -n vault-secrets-operator-system -l app.kubernetes.io/name=vault-secrets-operator
```

Reply:
Thank you, I updated it to use `vault-secrets-operator-controller-manager`